pkg/kubernetes/scripts/date.js

```diff
@@ -24,13 +24,31 @@
 
     angular.module('kubernetes.date', [])
 
+    .directive('refreshEveryMin', [
+        '$interval',
+        function($interval) {
+            return {
+                restrict: 'E',
+                link: function(scope, element, attrs) {
+                    var interval = $interval(function () {}, 60000);
```

Comment:
So does this schedule tons of different digests? If so, can we just schedule a single rootScope digest every minute?